azurelinuxagent/common/future.py

```diff
@@ -68,3 +76,24 @@ def get_linux_distribution_from_distro(get_full_name):
     full_name = distro.linux_distribution()[0].strip()
     osinfo.append(full_name)
     return osinfo
+
+def get_openwrt_platform():
+    """
+    Add this workaround for detecting OpenWRT products because
+    the version and product information is contained in the /etc/openwrt_release file.
+    """
+    result = [None, None, None]
+    openwrt_version = re.compile("^DISTRIB_RELEASE=['\"](\d+\.\d+.\d+)['\"]")
```

Comment:
```suggestion
    openwrt_version = re.compile(r"^DISTRIB_RELEASE=['\"](\d+\.\d+.\d+)['\"]")
```
This will avoid future issues with flake8 and Python 3.8.  Same for the next line.

Reply:
Good point, will solved this

Reply:
Fix have been done